src/widget/selectable_text.rs

```diff
@@ -416,13 +416,15 @@ fn draw<Renderer>(
     let State { paragraph, .. } = &state;
     let bounds = layout.bounds();
 
-    let x = match paragraph.horizontal_alignment() {
-        alignment::Horizontal::Left => bounds.x,
-        alignment::Horizontal::Center => bounds.center_x(),
-        alignment::Horizontal::Right => bounds.x + bounds.width,
+    let x = match paragraph.align_x() {
+        text::Alignment::Left => bounds.x,
+        text::Alignment::Center => bounds.center_x(),
+        text::Alignment::Right => bounds.x + bounds.width,
+        text::Alignment::Default => todo!(), // TODO: FIX
+        text::Alignment::Justified => todo!(), // TODO: FIX
```

Comment:
@tarkah Should we put `Default` together with `Left` arm since we dont have RTL text? And what about Justified?

Reply:
This is just a copy / paste from iced text, looks like both are the same as Left:

https://github.com/iced-rs/iced/blob/master/core%2Fsrc%2Fwidget%2Ftext.rs#L345